aequilibrae/project/zone.py

```diff
@@ -121,6 +121,8 @@ def connect_mode(
             mode_id=mode_id,
             link_types=link_types,
             connectors=connectors,
+            proj_nodes=self.project.network.nodes.data,
```

Comment:
This looks wrong. This basically loads ALL nodes each time a zone is instantiated